mmdet/datasets/cityscapes.py

```diff
@@ -263,8 +263,9 @@ def evaluate(self,
             # create CocoDataset with CityscapesDataset annotation
             self_coco = CocoDataset(self.ann_file, self.pipeline.transforms,
                                     None, self.data_root, self.img_prefix,
-                                    self.seg_prefix, self.proposal_file,
-                                    self.test_mode, self.filter_empty_gt)
+                                    self.seg_prefix, '.png',
```

Comment:
can we use self.seg_suffix here rather than simply hardcoding '.png'?

Reply:
Updated.